doc/changes.md

```diff
@@ -101,6 +101,7 @@
 
 ### Enhancements
 
+* **Breaking**: (Unneeded) extra quotation marks around text strings (containing whitespaces) are now considered as part of the text string ([#3132](https://github.com/GenericMappingTools/pygmt/pull/3132), [#3457](https://github.com/GenericMappingTools/pygmt/issues/3457))
```

Comment:
Be specific that it is double quotes `" "` that will show up?

```suggestion
* **Breaking**: (Unneeded) extra double quotes around text strings (containing whitespaces) are now considered as part of the text string ([#3132](https://github.com/GenericMappingTools/pygmt/pull/3132), [#3457](https://github.com/GenericMappingTools/pygmt/issues/3457))
```

Reply:
Hm. In principle both comibinations (`" ' ' " `, `' " " '`) are possible and working:

```python
import pygmt 

size = 1
title = "my title"

fig = pygmt.Figure()
fig.basemap(region=[-size, size] * 2, projection="X5c/1c", frame=[f'WSne+t"this is {title}"'])
fig.shift_origin(xshift="+w+1c")
fig.basemap(region=[-size, size] * 2, projection="X5c/1c", frame=[f"WSne+t'this is {title}'"])
fig.show()
```

![text_whitespace_quotationmarks_included](https://github.com/user-attachments/assets/7220cf11-e263-4169-b90d-8151994654ff)

Reply:
As shown in https://github.com/GenericMappingTools/pygmt/issues/3457#issuecomment-2376894897, single quotes are part of the text string since the first PyGMT version, so it's not considered to be a _breaking_ change in v0.12.0.

Reply:
Ah, OK. Then this is fair and makes sense.